R/generate_scenario_info.R

```diff
@@ -124,5 +133,69 @@ SCENARIO.INFO <- ScenarioInfo(aScenarioType = DEFAULT.SCENARIO.TYPE,
                               aLogitAgroForest = NA,
                               aLogitAgroForest_NonPasture = NA,
                               aLogitCropland = NA,
+                              aUseZeroCost = FALSE,
+                              aCalibrateShareWt = TRUE,
                               aScenarioName = paste0(DEFAULT.SCENARIO.TYPE, "_", "Perfect"),
                               aFileName = paste0(DEFAULT.SCENARIO.TYPE, "_", "Perfect"))
+
+
+#' update_scen_info
```

Comment:
@rplzzz: I created this because I kept doing these steps manually when I was testing the model -- specifically switching from hindcast to future, switching from calculated share weights to read in, etc. If you don't think it is useful for the broader audience, then we can just put it in the scripts folder. I can definitely add a note to the help file about what it does.